### PR TITLE
upgrade networking components

### PR DESCRIPTION
**Pull request by GoelDeepak**
## High Level Description

This patch fixes three issues:
1. Erlang crash whenever hostname was 64 characters long which disallowed spartan, navstar, minuteman from starting
2. Mesos_state parser error whenever network protocol or type was in capital. For instance, TCP/UDP or BRIDGE/HOST instead of tcp/udp or bridge/host
3. Minuteman issue in which backends were not removed when the tasks died if the backends were on overlay network 

## Related Issues

  - [DCOS-13478](https://jira.mesosphere.com/browse/DCOS-13478) Spartan crashes when calling 'set_hostname()' with long hostnames.
  - [DCOS-12154](https://jira.mesosphere.com/browse/DCOS-12154) Minuteman: Failed to parse task
  - [DCOS-13273](https://jira.mesosphere.com/browse/DCOS-13273) Failed to parse task error in navstar logs
  - [DCOS-13359](https://jira.mesosphere.com/browse/DCOS-13359) Backends are not removed from VIP table when killed

## Checklist for all PR's

  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated 
  - [x] Test Results: [link to CI job test results for component]
  - [x] Code Coverage (if available): [link to code coverage report]

List of PRs
  - https://github.com/dcos/navstar/pull/49
  - https://github.com/dcos/minuteman/pull/85
  - https://github.com/dcos/otp/pull/1
  - https://github.com/dcos/spartan/pull/31
  - https://github.com/dcos/lashup/pull/29
  - https://github.com/dcos/mesos_state/pull/24 
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and Review Process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using github's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

File: packages/navstar/buildinfo.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "sources": {
     "navstar": {
       "kind": "git",
-      "git": "https://github.com/dcos/navstar.git",
-      "ref": "4ef753e175a93adc597827aa28a836e43b8ccf3d",
+      "git": "https://github.com/GoelDeepak/navstar.git",
```
Comment (giubueno) [inline packages/navstar/buildinfo.json]: Do you really want to change it to your repo?

Comment (GoelDeepak) [inline packages/navstar/buildinfo.json]: Definitely not. This is just for the testing

**Comment by spahl** [comment]
@mesosphere-mergebot bump-ee

**Comment by spahl** [comment]
@GoelDeepak please fill out the PR template in the relevant sections for review

**Comment by spahl** [comment]
@aeyakovenko please review

**Comment by nlsun** [comment]
Shouldn't this also upgrade spartan to 19.2?

**Comment by aeyakovenko** [comment]
and networking-api in ee

**Comment by GoelDeepak** [comment]
@nsun done.
@aeyakovenko  yeah thats part of ee PR

**Pull request merged**